test/validation/api/classification/odp_classification_basic.c

```diff
@@ -707,7 +707,6 @@ static void cls_create_cos_with_hash_queues(void)
 
 	ret = odp_cls_capability(&capa);
 	CU_ASSERT_FATAL(ret == 0);
-	CU_ASSERT_FATAL(capa.hash_protocols.all_bits != 0);
```

Comment:
The commit message lacks any mention of this change. And this change looks wrong anyway, since this test function is run only when multiple hash queues are supported and that in turn implies that there must be at least one protocol field included in the hash.